workflow/snakemake_rules/curate.smk

```diff
@@ -20,30 +20,52 @@ Produces different output files for GISAID vs GenBank:
         metadata = "data/genbank/metadata_transformed.tsv"
         flagged_annotations = temp("data/genbank/flagged-annotations")
         duplicate_biosample = "data/genbank/duplicate_biosample.txt"
+    RKI:
+        fasta = "data/rki/sequences.fasta"
+        metadata = "data/rki/metadata_transformed.tsv"
 """
 
+
+rule transform_rki_data:
+    input:
+        ndjson="data/rki.ndjson",
+    output:
+        fasta="data/rki/sequences.fasta",
+        metadata="data/rki/metadata_transformed.tsv",
+    params:
+        subsampled=config.get("subsampled", False),
+    shell:
+        """
+        ./bin/transform-rki \
+            {input.ndjson} \
+            --output-fasta {output.fasta} \
+            --output-metadata {output.metadata}
+        """
+
+
 rule transform_biosample:
     input:
-        biosample = "data/biosample.ndjson"
```

Comment:
For future reference, it would be much cleaner to lump all snakefmt changes into a single commit separate from the feature changes. Currently, it's harder to review because the changes are mixed together.

Reply:
I've decided to refactor the commit history in the new PR and got rid of all snkfmt changes.

Unfortunately there's a bug in snakefmt that prevents me from applying snkfmt so we'll go without for now.